gspread/client.py

```diff
@@ -8,9 +8,11 @@
 """
 
 from http import HTTPStatus
-from typing import Type
+from typing import IO, Any, Dict, List, Mapping, Optional, Tuple, Type, Union
 
+from google.auth.credentials import Credentials
 from google.auth.transport.requests import AuthorizedSession
```

Comment:
last thing we can do here is using the special mypy comment to ignore these missing import: `# type: ignore` as a comment at the end of the line.

don't forget to format your code after that :wink:

Reply:
Should we do this for the requests package too?

Reply:
I don't see any issues with the `requests` package. Mypy apparently found the typing stubs